packages/ketcher-core/src/application/editor/operations/CanvasLoad.ts

```diff
@@ -32,7 +32,10 @@ export class CanvasLoad extends BaseOperation {
   execute(restruct: ReStruct) {
     const oldStruct = restruct.molecule
     restruct.clearVisels() // TODO: What is it?
```

Comment:
This clears up visuals elements of all Re* entities

Reply:
This comment is not mine - but, actually, for me is also it is not clear, why do we need to clear them